examples/typescript/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "sdkexamples",
```

Comment:
should this be `sdk-examples` or is it not following that naming pattern to allow for special handling (if needed) for `sdk-*` in the future?